lib/audited/audit.rb

```diff
@@ -49,7 +49,11 @@ class Audit < ::ActiveRecord::Base
     cattr_accessor :audited_class_names
     self.audited_class_names = Set.new
 
-    serialize :audited_changes, YAMLIfTextColumnType
+    if Rails.version >= "7.1"
```

Comment:
Might be better to use
```rb
Gem::Version.new(Rails.version) >= Gem::Version.new("7.1")
```
see
```rb
[8] pry(main)> Rails.version > "10.2.3"
=> true
[9] pry(main)> Gem::Version.new(Rails.version) >= Gem::Version.new("10.2.3")
=> false
```

Reply:
String version comparison is bad, agreed.

Reply:
Thank you for pointing this out!
See proposal for its resolution: https://github.com/collectiveidea/audited/pull/689